src/linker.py

```
@@ -0,0 +1,118 @@
+    try:
+        r = requests.get(WIKIPEDIA_API, params={
+            "action": "query", "list": "search", "srsearch": term, "format": "json", "srlimit": 1
+        }).json()
+        results = r.get("query", {}).get("search", [])
+        return results[0]["title"] if results else term
+    except:
+        return term
+
+def resolve_redirect(term):
+    """Resolves Wikipedia redirects (e.g. UK -> United Kingdom)."""
+    if len(term.split()) > 5 or term.isdigit(): return term
```

Ruff (0.15.0)

[error] Multiple statements on one line (colon)

(E701)